doc.css

```diff
@@ -103,7 +75,8 @@
 
```

Comment:
line no. 50 - .ow(better naming convention can be used - something like bg-ow or something similar)

Reply:
thanks for the suggestion Prachi, I'll try to keep adapt better css naming